File: sdks/java/src/test/java/org/nanoai/TestRuntime.java
package org.nanoai;

public class TestRuntime {
    public static void main(String[] args) {
        NanoRuntime runtime = new NanoRuntime();
        boolean loaded = runtime.loadModel("test.gguf");
        if (loaded) {
            System.out.println("Java: Model loaded");
        }
        String result = runtime.generate("Hello from Java");
        System.out.println("Java Result: " + result);
        runtime.close();
    }
}

File: sdks/rust/examples/test_runtime.rs
use nanoai::NanoRuntime;

fn main() {
    let runtime = NanoRuntime::new();

    if runtime.load_model("test.gguf", "default") {
        println!("Rust: Model loaded");
    }

    let result = runtime.generate("Hello from Rust", "default");
    println!("Rust Result: {}", result);
}
